apps/aecore/test/aecore_forking_SUITE.erl

```diff
@@ -167,300 +169,21 @@ check_sync_event(#{sender := From, info := Info} = Msg, Nodes) ->
 
 
 mine_one_block(N) ->
-    subscribe(N, block_created),
-    rpc_call(N, application, set_env, [aecore, aec_pow_cuckoo, {"mean16s-generic", "-t 5", 16}]), 
-    StartRes = rpc_call(N, aec_conductor, start_mining, []),
+    aecore_suite_utils:subscribe(N, block_created),
+    aecore_suite_utils:rpc_call(N, application, set_env, [aecore, aec_pow_cuckoo, {"mean16s-generic", "-t 5", 16}]), 
```

Comment:
Why is this not statically configured?

Reply:
Because the PoW can be configured by sys.config but not user config (yet - see https://www.pivotaltracker.com/story/show/153317000). And the current test suite code copies the sys.config without editing it.

Reply:
I like the approach to not have to copy files. This would require a bit of thought, but it would be less error prone if we could just start the code on 3 nodes with different settings without a config file... but I fear this is hard to do.

Reply:
For debugging, I think it's actually quite helpful to have config files to inspect in separate directories for each node.